README.md

```diff
@@ -1 +1,149 @@
+- It doesn't matter to which `customerName` 'program' each payload belongs to as long as `customerName` is the customer.
+
+**Example:**
+
+Considering we have the list of missions from this [data fixture][data-fixtures], and the following filter params:
+
+```js
+{
+  year: 2018,
+  customerName: "NASA"
+}
+```
+
+The expected result should be:
+
+```js
+[
+  {
+    flight_number: 62,
+    mission_name: "Iridium NEXT Mission 6",
+    payloads_count: 2,
+  },
+  {
+    flight_number: 72,
+    mission_name: "CRS-16",
+    payloads_count: 1,
+  },
+  {
+    flight_number: 64,
+    mission_name: "CRS-15",
+    payloads_count: 1,
+  },
+  {
+    flight_number: 60,
+    mission_name: "TESS",
+    payloads_count: 1,
+  },
+  {
+    flight_number: 59,
+    mission_name: "CRS-14",
+    payloads_count: 1,
+  },
+];
+```
+
+---
+
+### TASK #2 - React & Hooks
+
+Implement `RocketsList` component with the following specifications:
+
+1. It takes `filerParams` object (with shape described in [task #1][task-1]) as a prop
+2. It obtains a list of 'missions' from a [custom hook][custom-hook], which gets **the whole list of missions** from [SpaceX API][spacex-api] and processes them with `prepareData` function (from [task #1][task-1]) and `filterParams` prop
+3. For each 'mission' obtained from the custom hook, it renders a string using [template literals][template-literals] with the following format: "#`flight_number` `mission_name` (`payloads_count`)"
+4. While 'missions' are being fetched from API, it renders `"Loading..."` to screen
+5. If no 'missions' are obtained from the custom hook, it renders `"No data"` to screen
+
+**Observations:**
+
+- The list of missions should be re-processed if `filterParams` prop changes, **BUT no new API calls should happen** to [SpaceX API][spacex-api]
+- You are not allowed to use any of the filter parameters provided by the [SpaceX API docs][spacex-api-docs], just fetch all available data and process it with `prepareData` function
+
+**Example:**
+
+Considering we pass the following `filterParams` as a prop:
+
+```js
+{
+  year: 2018,
+  customerName: "NASA"
+}
+```
+
+The expected render should be:
+
+```txt
+#62 Iridium NEXT Mission 6 (2)
+#72 CRS-16 (1)
+#64 CRS-15 (1)
+#60 TESS (1)
+#59 CRS-14 (1)
+```
+
+## How to request help
+
+If you have any doubts, you can reach out to us by simply [creating a GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-repository) in your private repo.
+
+Describe your question(s) and [mention](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#mentioning-people-and-teams) [adjust-frontend-hiring][adjust-frontend-hiring] in your comments (don't forget to add user as collaborator). We will then receive a notification and get back to you as soon as possible.
```

Comment:
should be changed

Reply:
I didn't get the idea
Could you elaborate?

Reply:
that was a reminder to update if we'll drop this repo. Giving it a second read, I think it's meant to be their repo, so all good